ui/app/models/transform/template.js

```diff
@@ -22,7 +22,7 @@ const M = Model.extend({
     editType: 'regex',
     subText: 'The template’s pattern defines the data format. Expressed in regex.',
   }),
-  alphabet: attr('array', {
```

Comment:
Pulled down the branch and played around -- creation path works fine but it's a bit wonky when editing (see screenshot -- selected option is missing)
<img width="1432" alt="Screen Shot 2022-02-04 at 3 20 46 PM" src="https://user-images.githubusercontent.com/82459713/152604684-8480b6b9-ded8-4d52-b540-aaa02b6f7125.png">

Reply:
Ok thanks for pointing this out. I think I was looking at the response from the server and it is returned as a string so I was wondering why it would be an array attr but obviously there is a reason. My bad I'll revert.

Reply:
I reverted that change and all is well again :smile:. I found another bug while testing this where the decode formats data is disappearing on subsequent updates when the values are not changed which I am investigating.

Reply:
There seems to be something going on when using the `object` transform. If I remove it and don't use a transform on the `attr` everything works as expected.